src/ssz/type/container.zig

```diff
@@ -605,3 +631,28 @@ test "ContainerType - sanity" {
     _ = Foo.serializeIntoBytes(&f, f_buf);
     try Foo.deserializeFromBytes(allocator, f_buf, &f);
 }
+
+test "clone" {
+    const allocator = std.testing.allocator;
+    const Checkpoint = FixedContainerType(struct {
+        slot: UintType(8),
+        root: ByteVectorType(32),
+    });
+
+    var c: Checkpoint.Type = Checkpoint.default_value;
+
+    var cloned: Checkpoint.Type = undefined;
+    try Checkpoint.clone(allocator, &c, &cloned);
+    try std.testing.expect(&cloned != &c);
+    const Foo = VariableContainerType(struct {
+        a: FixedListType(UintType(8), 32),
+        b: FixedListType(UintType(8), 32),
+        c: FixedListType(UintType(8), 32),
+    });
+    var f = Foo.default_value;
+    defer Foo.deinit(allocator, &f);
+    var cloned_f: Foo.Type = undefined;
+    try Foo.clone(allocator, &f, &cloned_f);
+    try std.testing.expect(&cloned_f != &f);
```

Comment:
would be nice to test a cloned value has the same root + serialized data
same to other types